src/app/news/news-list.component.ts

```diff
@@ -82,6 +87,25 @@ export class NewsListComponent implements OnChanges {
       this.dialogsFormService.closeDialogsForm();
       this.dialogsLoadingService.stop();
     });
+    this.sendNewsNotifications(oldNews);
+  }
+
+  sendNewsNotifications(oldNews) {
+    const userId = this.userService.get()._id;
+    const link = '/community';
+    const notificationDoc = {
```

Comment:
We should also add `item: oldNews.replyTo` to this object so we can identify which message this is related to.